arbnode/sequencer_inbox.go

```diff
@@ -199,17 +199,17 @@ func (m *SequencerInboxBatch) getSequencerData(ctx context.Context, client *ethc
 			return nil, err
 		}
 
-		var blobInfo eigenda.EigenDABlobInfo
-		err = json.Unmarshal(certBytes, &blobInfo)
+		var eigenDACert eigenda.EigenDAV1Cert
+		err = json.Unmarshal(certBytes, &eigenDACert)
 		if err != nil {
 			return nil, err
 		}
 
 		arguments := abi.Arguments{
-			{Type: eigenda.DACertTypeABI},
+			addSequencerBatchFromEigenDACallABI.Inputs[1],
```

Comment:
index cert input type in `init`

Reply:
[1d29177](https://github.com/Layr-Labs/nitro/pull/72/commits/1d29177a44e6539176ddab696bd48ffc165bdab2)